flex/storages/rt_mutable_graph/mutable_csr.h

```diff
@@ -254,34 +490,33 @@ class MutableCsrBase {
 
 template <typename EDATA_T>
 class TypedMutableCsrConstEdgeIter : public MutableCsrConstEdgeIterBase {
-  using nbr_t = MutableNbr<EDATA_T>;
+  using nbr_t_ptr = typename MutableNbrSlice<EDATA_T>::nbr_t_ptr;
```

Comment:
I suggest indicating the const ptr in the type name, i.e. `using nbr_const_ptr_t`